questions/qa-ltr-scripts-in-rtl.en.html

```diff
@@ -392,7 +392,7 @@ <h2 lang="zh-hant" class="rlo notoc"><bdi class="lro">8.6%</bdi>受訪者沒頭
 <section id="styling_mirroring">
 <h3>Mirroring characters</h3>
 
-<p>If the font automatically reverses the images of the letters, that should take care of any necessary mirroring. However, the CSS transformation hack probably won't help, because it is only intended for use with short inline phrases. That said, it is perhaps less likely anyway that reversed letters are needed for scenarios that use the styled approach.</p>
+<p>If the font automatically reverses the images of the characters, that should take care of any necessary mirroring. However, the CSS transformation hack probably won't help, because it is only intended for use with short inline phrases. That said, it is perhaps less likely anyway that reversed characters are needed for scenarios that use the styled approach.</p>
```

Comment:
"it is perhaps less likely anyway" seems pretty wordy. Perhaps just "it is less likely"?

Reply:
@r12a what do you think?

Reply:
Because this is not directly related to the changes in this PR, I plan to merge this PR. Addison, if you think this needs to be changed, please submit a new PR. Thank you!